ksql-cli/src/main/java/io/confluent/ksql/cli/console/Console.java

```diff
@@ -14,12 +14,15 @@
 
 package io.confluent.ksql.cli.console;
 
+import static io.confluent.ksql.util.CmdLineUtil.splitByUnquotedWhitespace;
+import static io.confluent.ksql.util.CmdLineUtil.trimTrailingSemiColon;
```

Comment:
are there any style guidelines around static imports? IMO, it makes it a little harder to review code (I see it inline and I scroll around a little bit looking for the new method, get confused, then realize it's a static import)

Reply:
We don't AFAIK.  I generally review PRs in my IDE if they're of a decent size.   Makes it much easier to navigate around.